node/mock/blockProcessorStub.go

```diff
@@ -71,3 +71,7 @@ func (blProcMock BlockProcessorStub) DecodeBlockHeader(dta []byte) data.HeaderHa
 func (blProcMock BlockProcessorStub) AddLastNotarizedHdr(shardId uint32, processedHdr data.HeaderHandler) {
 	blProcMock.AddLastNotarizedHdrCalled(shardId, processedHdr)
 }
+
+func (blProcMock BlockProcessorStub) SetConsensusRewardAddresses([]string) {
+	panic("implement me")
```

Comment:
Should we panic in mocks?

Reply:
This was the model we adopted some time ago, in order to not care about the return values for methods that are not used but that need to be implemented for the unit/integration tests.